.devcontainer/Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM python:latest
+FROM python:3.10-alpine
```

Comment:
Alright, them it's better always use alpine images, it is a lightweight version of the images, and in python case it reduces from 341 mb to 18.61 mb

Reply:
I see. I guess [python:3.11-alpine](https://hub.docker.com/layers/library/python/3.11-alpine/images/sha256-1c7b5a998076ab7aa0a8745ab1461441a5bdc61e366985b9bfe3f4044c2b4503?context=explore) will do. The pro ject is created and modified on 3.11, and adds backward compatibility until 3.8. Is thnere any reason you selected 3.10 instead of 3.11?

Reply:
Edit: I just found out that few packages like git, weren't installed by default on alpine image. The `python:latest` by default comes with this dependencies preinstalled.

I don't think this is a good idea to go for alpine image.

Reply:
I see. Measuring the size of resulting Alpine image after adding dependencies comes close to vanilla image. So, there might not be much difference but rather make complicated to mantain dockerfile. For now, I think its better to go with vanilla image. Will think about it after resolving through dependencies in future.